### ISSUE TITLE
i3xrocks-app-launcher missing?

### ISSUE THREAD

**Issue by LocNgu** [gardotd426/regolith-de#46]
Thank you for this project.
When installing manually via makepkg or with AUR helper yay on a new arch build I always get this error message:
```
make[1]: Leaving directory '/home/user/regolith-de/src/i3xrocks'
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Removing static library files...
  -> Stripping unneeded symbols from binaries and libraries...
  -> Compressing man and info pages...
==> Checking for packaging issues...
==> Creating package "regolith-i3xrocks"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Starting package_i3xrocks-app-launcher()...
install: cannot stat '/home/user/regolith-de/src/regolith-i3xrocks-
config/scripts/app-launcher': No such file or directory
==> ERROR: A failure occurred in package_i3xrocks-app-launcher().
    Aborting...
```

Any idea what the issue might be?

**Comment by gardotd426** [comment]
Yep. They've moved to 2.0, and I am currently working on migrating everything to 2.0. But obviously I'm not doing that on the AUR PKGBUILD, I'm doing it on a separate branch. Everything was working fine because I was mostly using .deb packages, but the i3xrocks config was one of the few things where I was able to use a git repo (which is the preferred method). Well, they've removed the app launcher script from Regolith starting with 2.0. There will be a new app launcher, it just won't be through rofi (or i3xrocks), I believe Ken built one from scratch. 

Anyway, I just changed the PKGBUILD for the AUR version to check out the v1.6 branch of regolith-i3xrocks-config so it would be as it should be, until I've moved everything over. 

I've pushed the update. Close the issue if it builds correctly for you (as it should).
